examples/objective-c/auth_sample/AuthSample.xcodeproj/project.pbxproj

```diff
@@ -116,12 +116,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 63E1E9A21B28CB2100EF0978 /* Build configuration list for PBXNativeTarget "AuthSample" */;
 			buildPhases = (
+				A0E971D0BB7E77C68F1B71CA /* 📦 Check Pods Manifest.lock */,
```

Comment:
I recommend reverting the changes in the project files locally, before upgrading. Otherwise they might end up with both the emoji and the [CP] entries (as they are in this commit).